src/main/java/com/feed_the_beast/ftbu/cmd/CmdChunks.java

```diff
@@ -94,26 +99,30 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
             IForgePlayer p = FTBLibIntegration.API.getForgePlayer(player);
             ChunkDimPos pos = new EntityDimPos(player).toBlockDimPos().toChunkPos();
 
-            if(FTBUConfigWorld.CHUNK_CLAIMING.getBoolean())
+            if(!FTBUConfigWorld.CHUNK_CLAIMING.getBoolean())
             {
-                if(!p.equalsPlayer(ClaimedChunkStorage.INSTANCE.getChunkOwner(pos)) && !PermissionAPI.hasPermission(player.getGameProfile(), FTBUPermissions.CLAIMS_MODIFY_OTHER_CHUNKS, new BlockPosContext(player, pos.getChunkPos())))
-                {
-                    throw new CommandException("commands.generic.permission");
-                }
+                FTBLibIntegration.API.sendNotification(player, FTBUNotifications.CLAIMING_NOT_ENABLED);
+                return;
+            }
 
-                if(FTBUUniverseData.unclaimChunk(p, pos))
-                {
-                    FTBLibIntegration.API.sendNotification(player, FTBUNotifications.CHUNK_UNCLAIMED);
-                    updateChunk(player, pos);
-                }
-                else
-                {
-                    FTBLibIntegration.API.sendNotification(player, FTBUNotifications.CANT_MODIFY_CHUNK);
-                }
+            if(!FTBUtilitiesAPI_Impl.INSTANCE.hasPermission(player.getGameProfile(), FTBUPermissions.CLAIMS_CLAIM_CHUNKS, null)) {
```

Comment:
It doesn't work apparently. I even placed log traces, but it doesn't pick up implementation.